lib/ui/src/settings/about_page.tsx

```diff
@@ -19,16 +19,18 @@ class NotificationClearer extends Component<{ api: API; notificationId: string }
   }
 }
 
-export default () => (
+const AboutPage: FunctionComponent<{ onClose: () => void }> = ({ onClose }) => (
   <Consumer>
     {({ api }: Combo) => (
       <NotificationClearer api={api} notificationId="update">
         <AboutScreen
           current={api.getCurrentVersion()}
           latest={api.getLatestVersion()}
-          onClose={() => history.back()}
+          onClose={onClose}
         />
       </NotificationClearer>
     )}
   </Consumer>
 );
+
+export { AboutPage as default };
```

Comment:
since this was written we've switched to named exports (as opposed to default exports), so please export it as AboutPage and update the reference.